src/components/sbb-toast/sbb-toast.scss

```
@@ -0,0 +1,161 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-toast-z-index: var(--sbb-overlay-z-index);
+  --sbb-toast-max-width: #{sbb.px-to-rem-build(400)};
+  --sbb-toast-animation-duration: var(--sbb-animation-duration-6x);
+  --sbb-toast-animation-timing-function: ease;
+  --sbb-toast-color: var(--sbb-color-milk-default);
+  --sbb-toast-background-color: var(--sbb-color-midnight-default);
+  --sbb-toast-margin: var(--sbb-spacing-responsive-s);
+  --sbb-toast-padding-block: var(--sbb-spacing-responsive-xxxs);
+  --sbb-toast-padding-inline: var(--sbb-spacing-responsive-xs);
+  --sbb-toast-border-radius: var(--sbb-border-radius-4x);
+}
+
+:host([disable-animation]:not([disable-animation='false'])) {
+  --sbb-toast-animation-duration: 0s;
+}
+
+.sbb-toast__overlay-container {
+  position: fixed;
+  pointer-events: none;
+  inset: 0;
+  display: flex;
+
+  :host([data-state='closed']) & {
+    display: none;
+  }
+
+  :host([position='bottom-left']) &,
+  :host([position='bottom-start']) & {
+    justify-content: flex-start;
+    align-items: flex-end;
+  }
+
+  :host([position='bottom-center']) & {
+    justify-content: center;
+    align-items: flex-end;
+  }
+
+  :host([position='bottom-right']) &,
+  :host([position='bottom-end']) & {
+    justify-content: flex-end;
+    align-items: flex-end;
+  }
+
+  :host([position='top-left']) &,
+  :host([position='top-start']) & {
+    justify-content: flex-start;
+    align-items: flex-start;
+  }
+
+  :host([position='top-center']) & {
+    justify-content: center;
+    align-items: flex-start;
+  }
+
+  :host([position='top-right']) &,
+  :host([position='top-end']) & {
+    justify-content: flex-end;
+    align-items: flex-start;
+  }
+}
+
+.sbb-toast {
+  z-index: var(--sbb-toast-z-index);
```

Allow multilevel redefinition
```suggestion
  z-index: var(--sbb-toast-z-index, var(--sbb-overlay-z-index));
```